### DISCUSSION TITLE
[Question]: What's the best way to stop an object from being detected as a person?

### DISCUSSION THREAD

**Discussion by spikeygg** [blakeblackshear/frigate#10455] [Ask A Question] [answered]
### What is your question:

Frigate version: 0.13.2-6476F8A
Detector: Nvidia tensorrt (on RTX 3060)
Camera: Reolink RLC-410 (old-style, not 5MP)

I have a backyard camera and there's a post in my lawn for electricity that Frigate keeps thinking is a person. I get notifications that a person was detected in the backyard and it's a repeating picture of the pole. :(

For example:
![image](https://github.com/blakeblackshear/frigate/assets/1015891/73828d13-35a7-40a3-bf2d-583bc73d09ac)

When I look at the snapshot with the marker it looks like this:
![image](https://github.com/blakeblackshear/frigate/assets/1015891/a0f4788c-1821-45b6-96d1-66bce8bcd31e)

I'm leaning toward using the person-mask in that area to keep it from _seeing_ the pole but also left wondering if that's the best way to go about it... something like this:

```
    person:
      mask:
        - 2304,1147,2186,1144,2189,820,2304,816
```

Is this the best way? Anyone have a better solution for mitigating this mis-detection?

**Comment by NickM-27** [comment, marked as answer]
There is no universal best way, the best way depends on the specific false positive. 

In your case object mask is likely the best way since the sizing of the false positive is close to that of a real person and its position is static

**Reply by spikeygg** [reply]
okay, thanks for the quick response @NickM-27. I'll proceed with this.
